woocommerce/woocommerce-android
# Order details cant be opened from notification

Summary
==========
Fix issue #7483 by creating a global action to open the Order Details view with no attachment to the Order navigation graph. This global action will only be used when opening the Order Details from the notification, keeping the current strategy the same for everything else.

How to Test
==========
1. Open details of any Order
2. Create a new processing Order in wp-admin
3. Click on the received notification
3. Verify that details of the new Order open as expected

Update release notes:
